docs/orleans/overview.md

```diff
@@ -1,190 +1,133 @@
 ---
 title: Orleans overview
 description: An introduction to .NET Orleans.
-ms.date: 03/10/2022
+ms.date: 05/10/2022
 ---
 
 # Microsoft Orleans
 
-Orleans is a cross-platform framework for building robust, scalable distributed applications. It builds on the developer productivity of .NET and brings it to the world of distributed applications, such as cloud services. Orleans scales from a single on-premises server to globally distributed, highly-available applications in the cloud.
+Orleans is a cross-platform framework for building robust, scalable distributed applications. Distributed applications are defined as apps that span more than a single process, often beyond hardware boundaries using peer-to-peer communication. Orleans scales from a single on-premises server to globally distributed, highly-available applications in the cloud. Orleans extends familiar concepts and C# idioms to multi-server environments. Orleans is designed to scale elastically. When a host joins a cluster it can accept new activations. When a host leaves the cluster, either because of scale down or a machine failure, the previous activations on that host will be re-activated on the remaining hosts as needed. An Orleans cluster can be scaled down to a single host. The same properties which enable elastic scalability also enable fault tolerance. The cluster automatically detects and quickly recovers from failures.
 
-Orleans extends familiar concepts like objects, interfaces, `async` and `await`, and try/catch to multi-server environments. Accordingly, it helps developers experienced with single-server applications transition to building resilient, scalable cloud services and other distributed applications. For this reason, Orleans has often been referred to as "Distributed .NET".
+One of the primary design objectives of Orleans is to simplify the complexities of distributed application development by providing a common set of patterns and APIs. Developers familiar with single-server application development can easily transition to building resilient, scalable cloud-native services and other distributed applications using Orleans. For this reason, Orleans has often been referred to as "Distributed .NET" and is the framework of choice when building cloud-native apps. Orleans runs anywhere that .NET is supported. This includes hosting on Linux, Windows, and macOS. Orleans apps can be deployed to Kubernetes, virtual machines, and PaaS services such as [Azure App Service](/azure/app-service/overview) and [Azure Container Apps](/azure/container-apps/overview).
```

Comment:
Do you think the slide we did for MVP would help here? With the Lego boxes?

Reply:
@bradygaster - do you mean the illustration, that has grains drawn in horizontally amongst the controllers, hubs, views, and background services - stack?